db/migrate/20200328163747_add_premium_to_user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddPremiumToUser < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :premium_user, :boolean
```

Comment:
do we need to set default boolean to false as well for state management?

Reply:
added default false for the premium user